### PR TITLE
feat(scaffold-navigation): upgrade to bootstrap v4 and fontawesome v5

### PR DESCRIPTION
**Pull request by 3cp**
This has been a long requested upgrade.

### CODE REVIEW

**Comment by 3cp** [comment]
@shahabganji could you help to have a look on test_docer_2? I cannot reproduce that protractor error locally.

```
1) aurelia skeleton app should navigate to users page
  Message:
    Failed: element not interactable
```

**Comment by shahabganji** [comment]
@3cp Sure I will

File: skeleton/protractor/protractor.conf.js [hunk 1]
```diff
@@ -56,7 +56,8 @@ const config  = {
         '--disable-renderer-backgrounding',
         '--disable-device-discovery-notifications',
         '--no-sandbox',
-        '--disable-dev-shm-usage'
+        '--disable-dev-shm-usage',
+        '--window-size=1024,768'
```
Comment (3cp) [inline skeleton/protractor/protractor.conf.js]: @shahabganji this fix did the job. The reason for `element not interactable` is that the protractor browser window size is too small.

Nobody knows why the default screen size is so small inside docker(or maybe vm). Explicit window size does the trick.

Comment (3cp) [reply skeleton/protractor/protractor.conf.js]: When element is off screen, protractor says it's `not interactable`.

**Pull request merged**